src/alarm/alarm_manager.cc

```diff
@@ -54,7 +54,8 @@ int AlarmManager::ScheduleAlarm(const std::string& app_id,
   }
 
   int alarm_id = 0;
-  if (alarm->type() == AlarmInfo::ABSOLUTE) {
```

Comment:
Why is this part of the change better (by "part" I mean all changes to `alarm_manager.cc`)?